modules/xfem/src/efa/EFAEdge.C

```diff
@@ -406,3 +406,31 @@ EFAEdge::removeEmbeddedNode(EFANode * node)
     _intersection_x.erase(_intersection_x.begin() + index);
   }
 }
+
+bool
+EFAEdge::hasNode(EFANode * node)
+{
+  bool hasNode = false;
+
+  // check _edge_node1 for node
+  if (node == _edge_node1)
+    hasNode = true;
+
+  // check _edge_node2 for node
+  if (node == _edge_node2)
+    hasNode = true;
+
+  //  //check embedded nodes for node
+  //  for (unsigned int inode = 0; inode < _embedded_nodes.size(); ++inode)
```

Comment:
Are there no embedded nodes on the edge? If you don't need this code, delete it.